bot/menu.py

```diff
@@ -48,8 +49,12 @@ async def callback(self, interaction: Interaction):
             The action implemented by the user that needs to be notified.
             In the context of the bot, the action is a slash command
         """
-        for value in self.values:
-            add_subscriber(interaction.user.id, value)
+        if self.sub_toggle:
+            for value in self.values:
+                add_subscriber(interaction.user.id, value)
+        else:
+            for value in self.values:
+                remove_subscriber(interaction.user.id, value)

```

Comment:
Not familiar enough to question this directly, and it might be just the frontend background thinking about this, but putting a new list into the DB of subscriptions vs finding and removing individual subscriptions sounds cleaner

but up to you and @a-camarillo on this pattern i think